### PR TITLE
Add a note about the call {} in transaction for a detach delete sets transaction size 

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 1]
```diff
@@ -282,6 +282,10 @@ class CallSubqueryTest extends DocumentingTest {
           assertions = ResultAssertions(r => r.isEmpty),
           databaseStateBehavior = ClearState,
         ) { resultTable() }
+        note{
+          p("""The preceding usage of `IN TRANSACTIONS OF 2 ROWS` is an example given the data set described on the page.
```
Comment (tobias-johansson) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]:
```suggestion
          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.
```

**Review by tobias-johansson — CHANGES_REQUESTED** [review]
I have a suggestion for a different wording, WDYT?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala (hunk 1 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]: Much better, thanks @tobias-johansson!

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 2]
```diff
@@ -282,6 +282,10 @@ class CallSubqueryTest extends DocumentingTest {
           assertions = ResultAssertions(r => r.isEmpty),
           databaseStateBehavior = ClearState,
         ) { resultTable() }
+        note{
+          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.
+          #However, if you have a larger data set, you might want to use `IN TRANSACTIONS OF 10000 ROWS`.""".stripMargin('#'))
```
Comment (renetapopova) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]:
```suggestion
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 3]
```diff
@@ -282,6 +282,10 @@ class CallSubqueryTest extends DocumentingTest {
           assertions = ResultAssertions(r => r.isEmpty),
           databaseStateBehavior = ClearState,
         ) { resultTable() }
+        note{
+          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]:
```suggestion
          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.""".stripMargin('#'))
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 4]
```diff
@@ -282,6 +282,9 @@ class CallSubqueryTest extends DocumentingTest {
           assertions = ResultAssertions(r => r.isEmpty),
           databaseStateBehavior = ClearState,
         ) { resultTable() }
+        note{
+          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.""".stripMargin('#'))
+    }
```
Comment (renetapopova) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]:
```suggestion

```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala [hunk 5]
```diff
@@ -282,6 +282,8 @@ class CallSubqueryTest extends DocumentingTest {
           assertions = ResultAssertions(r => r.isEmpty),
           databaseStateBehavior = ClearState,
         ) { resultTable() }
+        note{
+          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.""".stripMargin('#'))
```
Comment (renetapopova) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala]:
```suggestion
          p("""Up to a point, using a larger batch size will be more performant. The batch size of `2 ROWS` is an example given the small data set used here. For larger data sets you might want to use larger batch sizes, like `10000 ROWS`.""".stripMargin)
```

**Pull request merged**